### ISSUE TITLE
Viewing single vendor privileges

### ISSUE THREAD

**Issue by plakitkelly** [keeleinstituut/tv-tolkevarav#570]
[Wiki](https://github.com/keeleinstituut/tv-tolkevarav/wiki/6.-Asutuse-peakasutaja-vaade#27-flow-view-vendors-database)
Preconditions:
User has the privilege "Teostajate andmebaasi vaatamine" (view_vendor_db)
2.7.2 User clicks specific user's "Vaata" button.

2.7.3 System displays a view where is data related to the vendor (name, email, phone, business name, tags, comment) and all the prices for specific language directions and skills (that relate to the specific user) and price rate percentages specific to the user.

The design is similar to [single user view](https://www.figma.com/file/VqCwED3Dymn7L1gGoAOK27/EKI-t%C3%B5lkev%C3%A4rav?node-id=757%3A93909&t=mgHRRWa7fNK7d4Ct-1), **but it's not possible to edit any of the vendor's data!**

Actual:
Currently it's not possible to view single vendor data with view_vendor_db privilege. If user clicks on Vaata button, user has navigated to main page and sees an error: Teil puudub ligipääs soovitud lehele.
User sees single vendor data with edit_vendor_db privilege.

**Comment by plakitkelly** [comment]
@KaarelKa It's still not possible to view single vendor's data with privilege view_vendor_db 
https://github.com/keeleinstituut/tv-tolkevarav/assets/130659408/7a3d37c4-3b0d-46d0-83a5-01d3f3816661

**Comment by KaarelKa** [comment]
@plakitkelly Yeah, sorry about that, this was left in 1 WIP branch which we ended up not merging

**Comment by plakitkelly** [comment]
It's OK
